.pipelines/android-daily.yml

```diff
@@ -0,0 +1,57 @@
+name: $(Year:yyyy).$(Month).$(DayOfMonth)-$(rev:r)
```

Comment:
why is this file here?

Reply:
This is the yml version of the Android daily pipeline. It will not delete the existing pipeline, but once it is checked into main, we can see if it is working as expected.